docs/website/docs/user-guides/quickstart.md

```diff
@@ -149,7 +149,54 @@ $ go mod init my.example.go.project
 
 Your source code has now been generated and created in the directory.
 
-## Step 1. Creating your application (`odo init`)
+
+## Step 1. Connect to your cluster and create a new namespace or project
+
+Before starting you should make sure that odo is connected to your cluster and that you have created a new namespace (or project if you are using OpenShift).
+
+
+
+
+<Tabs groupId="quickstart">
+  <TabItem value="kubernetes" label="Kubernetes">
+
+### Creating a new namespace
+
+If you are using Kubernetes, you can create a new namespace with the `odo create namespace` command.
+
+```console
+$ odo create namespace odo-dev
+ ✓  Namespace "odo-dev" is ready for use
+ ✓  New namespace created and now using namespace: odo-dev
+```
+
+  </TabItem>
+  <TabItem value="openshift" label="OpenShift">
+
+### Login to OpenShift Cluster
```

Comment:
maybe put this in a different tab and show kubernetes 1st? we want to try to remain Kubernetes #1 and show that first, especially with CNCF presentations / advertising / etc.

Reply:
I'm not sure I understand this. The Kubernetes is always the first tab, and the OpenShift is the second. Or did I miss it somewhere?

Reply:
Ah, I missed that part. I shouldn't be doing reviews on my day off, woops! When I went to preview it I hadn't seen it